pkg/controller/plan/migration.go

```diff
@@ -388,12 +392,16 @@ func (r *Migration) Cancel() error {
 
 	for _, vm := range r.Plan.Status.Migration.VMs {
 		if vm.HasCondition(Canceled) {
-			if err := r.cleanup(vm); err != nil {
-				r.Log.Error(err,
-					"Couldn't clean up after canceled VM migration.",
-					"vm",
-					vm.String())
+			dontFailOnError := func(err error) bool {
```

Comment:
couldn't it cause a spam in logs? if we will fail to clean up multiple times we will see the same message? the other option is your previous commit (boolean), but then we will need to check each function in cleanup to print the error. maybe the multiple message is ok

Reply:
it wouldn't spam the log too much since it's a one-time attempt but yeah, we'll have multiple logs with the same message but with different errors that should hint where we failed, that's a good thing in case several calls fail